lib/main/bindings/network/network_bindings.dart

```diff
@@ -91,6 +93,11 @@ class NetworkBindings extends Bindings {
   }
 
   void _bindingIsolateWorker() {
+    Get.put(Executor(), tag: BindingTag.threadExecutor);
```

Comment:
Why we need binding with tag?

Reply:
Need it to create other Executor for same type